fades/main.py

```diff
@@ -218,10 +225,16 @@ def go():
         logger.debug("Dependencies from source file: %s", indicated_deps)
         docstring_deps = parsing.parse_docstring(args.child_program)
         logger.debug("Dependencies from docstrings: %s", docstring_deps)
-    reqfile_deps = parsing.parse_reqfile(args.requirement)
-    logger.debug("Dependencies from requirements file: %s", reqfile_deps)
+
+    reqfile_deps = []
+    for rf_path in args.requirement:
+        rf_deps = parsing.parse_reqfile(rf_path)
+        logger.debug('Dependencies from requirements file "%s": %s', rf_path, rf_deps)
+        reqfile_deps.append(rf_deps)
+
     manual_deps = parsing.parse_manual(args.dependency)
     logger.debug("Dependencies from parameters: %s", manual_deps)
+
     indicated_deps = _merge_deps(ipython_dep, indicated_deps, docstring_deps,
```

Comment:
Let's have a list around for all deps and that's it. This way we could call `_merge_deps(all_dependencies)`.

So start with an `all_dependencies` above and add the different dependencies from the different sources. One thing to pay attention is the order how these dependencies are stored in the list, it should respect:

1. ipython dep
1. deps from source file 
1. deps from file docstring
1. deps from requirement files
1. deps manually specified with `-d`